.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.5
```

Comment:
Ruby 3.0.0 is out now. Should be using the latest current version of Ruby since this is a new project?

Reply:
I tried 2.7.x but ran into issues (IIRC with circle). Hadn't thought about 3.x, but probably will be similar. Thought best plan was to get it running in CP on 2.6.x, and worry about upgrading after that.